dll/cpl/appwiz/createlink.c

```diff
@@ -265,7 +265,7 @@ WelcomeDlgProc(HWND hwndDlg,
                     PathRemoveBackslashW(pContext->szWorkingDirectory);
                     pch = PathFindFileNameW(pContext->szWorkingDirectory);
                     if (pch && *pch)
-                        *pch = 0;
```

Comment:
Copying the file name into `szDescription` makes sense. But why are you removing the `*pch = 0`, which removes the file name from `szWorkingDirectory`?

Reply:
Yes! You're righ.